integration/commands_administration_test.go

```diff
@@ -1012,6 +1013,26 @@ func TestCommandsAdministrationServerStatus(t *testing.T) {
 	assert.Equal(t, int32(0), must.NotFail(catalogStats.Get("timeseries")))
 	assert.Equal(t, int32(0), must.NotFail(catalogStats.Get("views")))
 	assert.Equal(t, int32(0), must.NotFail(catalogStats.Get("internalViews")))
+
+	t.Skip("https://github.com/FerretDB/FerretDB/issues/2447")
+	opts := &options.CreateCollectionOptions{
```

Comment:
`opts := options.CreateCollection()`, `opts.SetCapped(true)`, etc. Something we do in (almost) all other tests